### PR TITLE
Updates to tickets/DM-25869

### PR DESCRIPTION
**Pull request by SimonKrughoff**
This was branched from the DM-25869 ticket branch.  It is intended to be a set of suggested changes largely to clarify the current state of EFD deployments and also what will be the case when summit activities resume.

### CODE REVIEW

File: Control-User-Interfaces/EFD.rst [hunk 1]
```diff
@@ -34,9 +40,16 @@ Chronograf
 .. note::
 
   To login to Chronograf, you need to be a member of the GitHub lsst-sqre organization. If you are not, please request access on the ``#com-square`` LSSTC Slack channel.
+  You will also need access to the appropriate VPN to log in.
+  For information on the US data facility (NCSA) VPN see `instructions here <https://wiki.ncsa.illinois.edu/display/cybersec/Virtual+Private+Network+(VPN)+Service#VirtualPrivateNetwork(VPN)Service-UsingtheCiscoAnyConnectVPNClient(Required)>`_.
+  For information about connecting to the Chilean VPN, please ask on the ``#rubinobs-it-chile`` LSSTC Slack channel.
 
 .. Would be nice to show some sort of screenshot here?
```
Comment (patrickingraham) [inline Control-User-Interfaces/EFD.rst]: You can remove this now :)

File: Control-User-Interfaces/EFD.rst [hunk 2]
```diff
@@ -51,84 +64,26 @@ It contains multiple methods to aid with queries and data interactions, essentia
...
 
-The following sections list the EFD instances available, where they run and who the intended audience is for each one.
+Following is a list of EFD instances.
+Please refer https://sqr-034.lsst.io for technical details: e.g. information about connecting directly to the influxDB server API.
+There are also several instances of the EFD deployed at various places that are unlikely to be of general interest.
+These are also listed in the aforementioned SQuaRE technical note.
+
+The entry titled "efd_client alias" is the string to pass to the `EfdClient` constructor in order to authenticate to that instance of the EFD.
 
 ..  I just 95% copied/pasted here, but I think only the summit, base and LSP instances are required to be shown?
```
Comment (patrickingraham) [inline Control-User-Interfaces/EFD.rst]: same here, this was just to help guide you...

File: Control-User-Interfaces/EFD.rst [hunk 3]
```diff
@@ -34,9 +40,16 @@ Chronograf
 .. note::
 
   To login to Chronograf, you need to be a member of the GitHub lsst-sqre organization. If you are not, please request access on the ``#com-square`` LSSTC Slack channel.
+  You will also need access to the appropriate VPN to log in.
+  For information on the US data facility (NCSA) VPN see `instructions here <https://wiki.ncsa.illinois.edu/display/cybersec/Virtual+Private+Network+(VPN)+Service#VirtualPrivateNetwork(VPN)Service-UsingtheCiscoAnyConnectVPNClient(Required)>`_.
+  For information about connecting to the Chilean VPN, please ask on the ``#rubinobs-it-chile`` LSSTC Slack channel.
 
 .. Would be nice to show some sort of screenshot here?
 
+.. figure:: /weather.png
```
Comment (patrickingraham) [inline Control-User-Interfaces/EFD.rst]: This figure isn't showing up in the html render. I can fix it if you'd like though.

**Review by patrickingraham — APPROVED** [review]
Looks good.
I made 2 comments about removing comments before realizing you made your own branch and were merging it to my ticket branch. 
One other comment about the figure not working, but I can probably fix it if you'd like.
Feel free to ignore the comments and merge.
Thanks!

**Comment by SimonKrughoff** [comment]
There are couple of rendering issues I'd like to take care of before merging.  I'll try to do those now.

**Comment by SimonKrughoff** [comment]
OK rendering issues resolved and comments removed.  I'm going to merge as soon as the checks pass and I can look at the render.

**Pull request merged**